frontend/src/services/NotificationService.ts

```
@@ -0,0 +1,100 @@
+    try {
+      await axios.post(`${apiURL}/notification/${userId}/${notificationId}`);
+    } catch (error) {
+      const message =
+        axios.isAxiosError(error) && error.response?.data?.error
+          ? error.response.data.error
+          : "Failed to mark notification as read";
+
+      return new Error(message);
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)